modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-order-impl/src/main/java/com/liferay/headless/commerce/admin/order/internal/resource/v1_0/OrderItemResourceImpl.java

```diff
@@ -288,6 +288,155 @@ public OrderItem postOrderIdOrderItem(Long id, OrderItem orderItem)
 			_commerceOrderService.getCommerceOrder(id), orderItem);
 	}
 
+	@Override
+	public OrderItem putOrderItem(Long id, OrderItem orderItem)
+		throws Exception {
+
+		CommerceOrder commerceOrder = _commerceOrderService.getCommerceOrder(
+			orderItem.getOrderId());
+
+		CommerceOrderItem commerceOrderItem =
+			_commerceOrderItemService.fetchCommerceOrderItem(id);
+
+		commerceOrderItem = _commerceOrderItemService.updateCommerceOrderItem(
```

Comment:
As in the definition are we only using update or upsert? It is not clear between ticket, schema description and what I see here

Reply:
Changed description